components/Chat/AutocompleteChat.vue

```diff
@@ -120,8 +120,8 @@
   .chat-autocomplete {
     position: absolute;
     top: 0;
-    transform: translate( 12.5%, -100% );
-    width: 75%;
+    transform: translate( 0%, -100% );
+    width: 90%;
```

Comment:
the X value should be half of 100 - width value, so it's centered.
So shouldn't this be translated 5% instead of 0?

Reply:
fixed